docs/DeveloperGuide.adoc

```diff
@@ -279,104 +298,446 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* money-conscious NUS students
```

Comment:
As a suggestion, you can consider widening the scope of the product if the functionality can be extended to a wider target audience without much additional hassle, e.g. to students in general.